SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security
+
+Hathor Labs has a bounty program to encourage white hat hackers to collaborate in identifying security breaches and vulnerabilities in Hathor core. To know more about this, see [https://immunefi.com/bounty/hathornetwork/](https://immunefi.com/bounty/hathornetwork/).
```

Comment:
I think we should have a simple page about the Bounty Program in Hathor's website. So all links point there. This page will have a link to Immunefi or any other company handling our bug bounty programs.

Reply:
Proposal: keep as is.
Reasons:
- Currently the complete description of the bug bounty program can be found on Immunify.
- Every place where the subject "security/bug bounty" is mentioned, Immunify is mentioned. This already occurs in the following locations:
- Main website: https://hathor.network/developers/ (help us improve section)
- Documentation website: https://docs.hathor.network/references/besides-documentation#security
- And now, the proposal is to do the same in all READMEs in public repositories.

- It is good practice to have a "security" doc in each repository. To an outsider, it denotes a high level of quality in our codebase.

- In this context, we would have to create a new section, albeit small, perhaps repeating what is already in the `SECURITY.md` doc or adding content that is in Immunify.
- In both cases, there does not seem to be any benefit to this. On the contrary, it would make updating the content even more difficult.

- The only advantage seems to be being able to update a single link, when in the future the bug bounty program migrates from Immunify to another platform.
- However, the cost of changing links in markdowns and website is marginal.
- Furthermore, in any case the best solution would not be to add a new section to the website, but rather to make a note using the DNS of the hathor.network domain.

Now: if you understand that it is important to have more information about security than we currently have on the website, the solution would be to keep this PR as is, and add the content of `SECURITY.md` on the hathor.network website, just as we have on https ://docs.hathor.network/references/besides-documentation#security.